openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -71,3 +71,36 @@ export function initEdit() {
     update_len();
     show_hide_title();
 }
+
+/**
+ * Initializes links element on edit page.
+ *
+ * Assumes presence of elements with id:
+ *    - '#links'
+ *    - '#link-label'
+ *    - '#link-url'
+ *    - '#link-errors'
+ */
+export function initEditLinks() {
+    $('#links').repeat({
+        vars: {
+            prefix: '$prefix'
+        },
+        validate: function(data) {
+            if ($.trim(data.url) == '' || $.trim(data.url) == 'http://') {
```

Comment:
Let's make these check triple equals ===

Note to self, should this be checking https:// now we use https ?

Reply:
I'll update to `https://` as that's the default value from web.html and this only checks if the user entered something.
Maybe we can make this a variable in the template and send it to JS.